README.md

```diff
@@ -64,6 +64,9 @@ Then, back in vuu-ui, run the launch script"
 npm run launch:demo:electron
 ```
 
+### Workspace Settings 
+A set of VSCode settings is shared as workspace settings. In order to use the workspace settings the project needs to be opened in VSCode from vuu-ui/ folder 
+
```

Comment:
I would expand this to mention what it is the VSCode workspace settings is providing so other IDE users can match VSCode. I'd also mention the intelliJ/Java settings as well.

Reply:
I have added the list of settings provided by the workspace. I don't think the intelliJ/Java settings should be in scope for this change

Reply:
How come?

Reply:
I have added IntelliJ Settings for Java as shown in the screenshot on the ticket